deployment/ansible/roles/cac-tripplanner.app/tasks/dev-test-dependencies.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: Install dev/test python packages
+  pip: name={{ item.name }} version={{ item.version }}
+  with_items:
+    - { name: 'flake8', version: '2.3.0' }
```

Comment:
Just had a (very) brief conversation with @notthatbreezy about installing dev/test deps separately. Might want to check with him regarding this.